README.md

```diff
@@ -143,7 +147,9 @@ The supported *request specific options* are:
   asStream: boolean, // default `false`
   compression: string, // default `false`
   headers: object, // default `null`
-  querystring: object // default `null`
+  querystring: object // default `null`,
+  context: object // default `null`,
+  id: any // default incr. integer
```

Comment:
I think it would make sense to use the `unknown` type here, but it's only available in TS 3.0+, which is about a year old...

Reply:
I didn't know about `unknown`, but I'm not sure it would make sense here.
I think that `any` is the most appropriate since I'm not doing anything with that but passing it around. The user can choose literally any value :)

Reply:
yeah, `unknown` has the same effect, but where it's different is when you/someone tries to use it internally. `any` will allow any operation such as `meta.request.id.toString()` works on `any`, but with `unknown` the type check will fail because there wasn't a check to make sure that `id` has a `toString` function.

As I type this though I realize that only makes a difference if the internals are implemented in TypeScript, which they aren't, so it makes sense to go for the most widely supported option (`any`).